sklearn/tests/test_docstring_parameters.py

```diff
@@ -32,8 +32,10 @@
 with warnings.catch_warnings():
     warnings.simplefilter('ignore', FutureWarning)
     PUBLIC_MODULES = set([
-        pckg[1] for pckg in walk_packages(prefix='sklearn.',
-                                          path=sklearn.__path__)
+        pckg[1] for pckg in walk_packages(
+            prefix='sklearn.',
+            # mypy error: Module has no attribute "__path__"
+            path=sklearn.__path__)  # type: ignore
```

Comment:
Why is this causing a typing error?

Reply:
It's a known mypy issue https://github.com/python/mypy/issues/1422. For now the the workaround is to ignore it. I'll add an inline comment.